### PR TITLE
support control signals in MusicVAE

### PR DESCRIPTION
**Pull request by iansimon**

### CODE REVIEW

File: music_rnn/src/model.ts [hunk 1]
```diff
@@ -52,6 +52,8 @@ export class MusicRNN<A extends magenta.controls.ControlSignalUserArgs> {
    * `NoteSequence` and `Tensor` objects. If not provided, a `converter.json`
```
Comment (adarob) [inline music_rnn/src/model.ts]: Please add (Optional) here too for consistency.

Comment (adarob) [inline music_rnn/src/model.ts]: Can you add an example to the demo either here or in a follw-up PR?

Comment (iansimon) [reply music_rnn/src/model.ts]: I'll do another PR where I add examples with chord-conditioning in both music_rnn and music_vae.

Comment (iansimon) [reply music_rnn/src/model.ts]: Done.

**Pull request merged**